ios/RNDemo/NativeMobileSDKBridge.m

```diff
@@ -142,6 +142,31 @@ @implementation NativeMobileSDKBridge
 }
 
 #pragma mark: Media Related Function
+-(void)configureActiveAudioDevice:(NSArray<MediaDevice *> * _Nonnull)audioDeviceList
+{
+
+  NSArray<AVAudioSessionPortDescription *> *currentOutputs = [[[AVAudioSession sharedInstance] currentRoute] outputs];
```

Comment:
Hi I think I understand the logic for this. I guess this this is because we don't provide priority like Android.

Any reason why you don't use `MediaDevice.port` and instead directly calling system API?

There could be few more scenarios that we should think of. 
1. Bluetooth connected
2. Airplay connected
3. Headphone connected

For basic use case, I think we can start with 
Bluetooth
Headphone.

 so code might look like this
```
bluetoothDevice = nil
headphoneDevice = nil

// loop through audioDeviceList
if audioDeivce.port == bluetooth -> set bluetoothdevice
if headphoneDeivce -> set headset

if bluetoothDevice != nil
 chooseAudioDevice(bluetooth)
else if headphoneDevice != nil
  chooseAudioDevice(headphone)
else
   chooseAudioDevice(audioDeviceList[1]) // we know that if there is speaker we insert to index 1 or you can find speaker device.
```
This is pseudo code , but hope you got the idea.

Reply:
@hyandell 
[listAudioDevices](https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/device/DefaultDeviceController.swift#L40) is implemented by `AVAudioSession.availableInputs` .
It means `listAudioDevices` return only input devices list.  AFAIK, We cannot get headphones connection status through `MediaDevice.port`.　
I tried to implement by `MediaDevice.port`, but it didn't work.


> There could be few more scenarios that we should think of.
Sure, this idea is great. 
But I think there are already implemented cheap edition in this PR.
`listAudioDevices` is return fixed order devices list.
In most case. Behavior of 'choice last device' is expected by the user. without 'using wired headphones case'


(eg. connecting bt headset → use bt headset)

I think it is enough for to resolve this issue #5 .
And that works for My app.

Reply:
Hey @KONDO-Yuuki this is amazing <3 I think this will help us a lot.
Just wanted to ask, even if it is taking a bit of time, if I just copy you changes, I would have in speaker by default right?